internal/state/enumerated_total/tree.go

```diff
@@ -0,0 +1,204 @@
+package enumerated_total
```

Comment:
Rename the package to `enumeratedtotal`, as it's Go's convention https://google.github.io/styleguide/go/decisions#package-names

Reply:
changed